### ISSUE TITLE
Additional form field declared in subclass of EntangledModelForm does not work anymore

### ISSUE THREAD

**Issue by AlexanderB-C** [jrief/django-entangled#28]
My `class DeliveryNoteLineItemForm(forms.ModelForm)` already had a `package_number = forms.CharField()` additional field declared, which does not actually exist in the DeliveryNoteLineItem model.

It is used in the `def form_valid(self, form):` method of my `generic.CreateView` based class based view to do some custom logic before actually saving the new object to the database.

This field is not included in the `form.cleaned_data` dictionary anymore now that I switched the form class to inherit from `EntangledModelForm`, declared more additional form fields for use in `entangled_fields`, etc.

The `package_number` field is not included in the `entangled_fields`, so it should still be part of `form.cleaned_data` as a completely stand alone field in my opinion.

**Comment by fsbraun** [comment]
Have you included it in`untangled_fields` (or `fields`)?

**Comment by AlexanderB-C** [comment]
No, as in `fields` you are only supposed to put fields of the model, or am I wrong there?
It was not in `fields` before I switched to `EntangledModelForm` either but worked like this.

**Comment by fsbraun** [comment]
Great to hear it works for now. I'll have a look into the matter.
